packages/dds/tree/src/simple-tree/api/simpleSchema.ts

```diff
@@ -125,19 +125,11 @@ export interface SimpleFieldSchema {
  *
  * @sealed
  */
-export interface SimpleTreeSchema {
+export interface SimpleTreeSchema extends SimpleFieldSchema {
 	/**
 	 * The complete set of node schema definitions recursively referenced by the tree's {@link SimpleTreeSchema.allowedTypes}.
 	 *
 	 * @remarks the keys are the schemas' {@link TreeNodeSchemaCore.identifier | identifiers}.
 	 */
 	readonly definitions: ReadonlyMap<string, SimpleNodeSchema>;
-
-	/**
-	 * The types allowed under the root of the tree.
```

Comment:
This doc seems more appropriate for SimpleTreeSchema than the version it now gets from SimpleFieldSchema. We can keep the redundant definition here to give it more contextually relevant docs, right? If so, maybe do the same think for `kind`.

Reply:
Fair. I'll update.